modules/test/playwright/tests/object-web/main/objectView.spec.ts

```diff
@@ -256,3 +256,76 @@ test('assert that the user is able to use the ERC field in Sort, on the Custom V
 
 	await expect(page.getByRole('cell').nth(2)).toHaveText(entry2);
 });
+
+test('can create an object view column and it still stays in order', async ({
+	apiHelpers,
+	objectViewPage,
+	page,
+}) => {
+	const objectDefinition =
+		await apiHelpers.objectAdmin.postRandomObjectDefinition({
+			objectFolderExternalReferenceCode: 'default',
+			status: {code: 0},
+		});
+
+	apiHelpers.data.push({id: objectDefinition.id, type: 'objectDefinition'});
+
+	await objectViewPage.goto(objectDefinition.label['en_US']);
```

Comment:
Why go to `objectViewPage` before creating the view? Move this to after calling `postObjectDefinitionObjectView`.